### PR TITLE
feat: add docker

### PR DESCRIPTION
**Pull request by DonMagee**
Docker has a new pricing model that charges extra for SSO.

Source: https://www.docker.com/blog/updating-product-subscriptions/
and https://www.docker.com/pricing

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -75,6 +75,14 @@
   pricing_source: https://coderpad.io/pricing
   updated_at: 2019-06-28
 
+- name: Docker
+  url: https://www.docker.com/
+  base_pricing: $7 per u/m
+  sso_pricing: $21 per u/m
+  percent_increase: 33%
```
Comment (w0de) [inline _data/vendors.yml:82]: This is a 300% increase (if one isn't pedantic with per _cent_ but that's another review)

**Review by w0de — CHANGES_REQUESTED** [review]
Was halfway through adding my own PR for Docker when I saw yours. Figured I'd help!

File: _data/vendors.yml [hunk 2]
```diff
@@ -75,6 +75,14 @@
   pricing_source: https://coderpad.io/pricing
   updated_at: 2019-06-28
 
+- name: Docker
+  url: https://www.docker.com/
+  base_pricing: $7 per u/m
+  sso_pricing: $21 per u/m
```
Comment (w0de) [inline _data/vendors.yml:81]:
```suggestion
  sso_pricing: $21 per u/m
  footnotes: "[^docker]: SSO "coming soon" as of 08/21 so you may just be paying for the promise!"
```

File: _data/vendors.yml (hunk 1 above)
Comment (w0de) [inline _data/vendors.yml:82]:
```suggestion
  percent_increase: 300%
```

**Comment by lcycon** [comment]
Fixed in #273

**Pull request closed without merging**